docs/messaging.rst

```diff
@@ -1312,9 +1338,21 @@ Debug event
 This message type is used by debugging kernels to send debugging events to the
 frontend.
 
+The content of the debug events follows the schema of the ``Event`` message of
+the *Debug Adapter Protocol* (DAP_).
+
 Message type: ``debug_event``::
 
-    content = {}
+    content = {
+        # The type of debug message
+        'type': 'event',
+        # A unique sequence number
```

Comment:
Does the DAP mention anything about `seq` being unique?

Reply:
The DAP says:

```
Message Sequence (also known as message ID). For protocol messages of type 'request' this ID can be used to cancel the request
```